pytorch_lightning/trainer/trainer.py

```diff
@@ -894,6 +891,28 @@ def enable_validation(self) -> bool:
         val_loop_enabled = (self.is_overridden('validation_step') and self.limit_val_batches > 0)
         return val_loop_enabled or self.fast_dev_run
 
+    @property
+    def default_root_dir(self) -> str:
+        """
+        The default location to save artifacts of loggers, checkpoints etc.
+        It is used as a fallback if logger or checkpoint callback do not define specific save paths.
+        Defaults to ``os.getcwd()``.
+        """
+        path = self._default_root_dir or os.getcwd()
```

Comment:
I would rather this have in init and write to log a warning that no part given so we use the actual folder

Reply:
ok i moved it to init, but I don't like it as much because of documentation. I disagree with warning, because it is a default and it is documented and we don't print warnings for other trainer defaults.